.circleci/config.yml

```diff
@@ -1,15 +1,41 @@
+save_cache: &save_cache
+  save_cache:
+    key: yarn-deps-{{ checksum "yarn.lock" }}
+    paths:
+      - ./node_modules
+
+restore_cache: &restore_cache
+  restore_cache:
+    keys:
+    - yarn-deps-{{ checksum "yarn.lock" }}
+    - yarn-deps-
```

Comment:
What is this second key about?

Reply:
It's a fallback. CircleCI has an immutable caching structure. It saves a cache with an associated key and will never change that version of the cache after the first write. When reading from the cache, it _always_ does a partial string match. If it doesn't find a match to the first key, it'll fall over to the next. That means in this case it'll load the last version of the cache.

Reply:
Nice